specification/at_protocol_specification.md

```diff
@@ -694,81 +694,81 @@ data:
 }
 ```
 
-If the other Secondary Server on which the lookup needs to be performed is down then the secondary should return the following error and keep the connection alive.
+If the other atServer on which the lookup needs to be performed is down then the atServer should return the following error and keep the connection alive.
 
 
-```error:AT0007-Secondary Server not found.```
+```error:AT0007-atServer not found.```
```

Comment:
The code hasn't been updated yet, so it still says `Secondary server not found`

Reply:
I've raised https://github.com/atsign-foundation/at_tools/issues/275 to get this changed, but that might not happen straight away.

Reply:
The docs here need to reflect the actual error message until it gets fixed in the code.